src/libs/colorpicker.c

```diff
@@ -164,8 +164,14 @@ static void _update_sample_label(dt_colorpicker_sample_t *sample)
 
     case 2:
       // LCh
-      dt_Lab_2_LCH(lab, lch);
-      snprintf(text, sizeof(text), "%.02f %.02f %.02f", CLAMP(lch[0], .0f, 100.0f), lch[1], lch[2] * 360);
+      dt_Lab_2_LCH(lab, alt);
+      snprintf(text, sizeof(text), "%.02f %.02f %.02f", CLAMP(alt[0], .0f, 100.0f), alt[1], alt[2] * 360);
+      break;
+
+    case 3:
+      // HSL
+      dt_RGB_2_HSL(rgb, alt);
+      snprintf(text, sizeof(text), "%.02f %.02f %.02f", alt[0], alt[1], alt[2]);
```

Comment:
The "H" channel (alt[0]) of HSL should be multiplied to 360 just like the "h" channel of LCh.